rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -212,6 +215,39 @@ def add_recording(
         if blueprint is not None:
             recording.send_blueprint(blueprint)
 
+    def _add_table_id(self, record_batch: RecordBatch, table_id: str) -> RecordBatch:
+        # Get current schema
+        schema = record_batch.schema
+        schema = schema.with_metadata({b"__table_id": table_id})
+
+        # Create new record batch with updated schema
+        return RecordBatch.from_arrays(record_batch.columns, schema=schema)
+
+    def send_table(
+        self,
+        id: str,
+        table: RecordBatch,
+    ) -> None:
+        """
+        Sends a table in the form of a dataframe to the viewer.
+
+        Parameters
+        ----------
+        id : str
+            The name that uniquely identifies the table in the viewer.
+            This name will also be shown in the recording panel.
+        table : RecordBatch
```

Comment:
no need to repeat the types
```suggestion
        id:
            The name that uniquely identifies the table in the viewer.
            This name will also be shown in the recording panel.
        table:
```